src/main/java/com/github/fge/jsonpatch/DualPathOperation.java

```diff
@@ -48,8 +57,9 @@
     protected DualPathOperation(final String op, final JsonPointer from,
         final JsonPointer path)
     {
-        super(op, path);
+        this.op = op;
```

Comment:
Ah I didn't realize it was used in serialization.   Good call.
On Sep 23, 2015 7:47 AM, "hnguyen08" <notifications@github.com> wrote:

> In src/main/java/com/github/fge/jsonpatch/DualPathOperation.java
> <https://github.com/box-metadata/json-patch/pull/2#discussion_r40211864>:
>
> > @@ -48,8 +57,9 @@
> >      protected DualPathOperation(final String op, final JsonPointer from,
> >          final JsonPointer path)
> >      {
> > -        super(op, path);
> > +        this.op = op;
>
> Let's keep it for now and remove it in a different change. It is useful
> for serialization of all "dual-path" operations, but we can consider
> serializing in a different way (@JsonProperty annotations?).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/box-metadata/json-patch/pull/2/files#r40211864>.
>